src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Arkivmelding.cs

```diff
@@ -0,0 +1,412 @@
+using System;
+using System.Xml.Serialization;
+
+namespace Altinn.Common.EFormidlingClient.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ArkivmeldingMetadata"/> class.
+    /// </summary>
+    /// 
+    [XmlRoot(ElementName = "arkivmelding")]
+    public class Arkivmelding
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ArkivmeldingMetadata"/> class.
+        /// </summary>
+        /// <param name="system">System</param>
+        /// <param name="meldingId">MeldingId</param>
+        /// <param name="tidspunkt">Tidspunkt</param>
+        /// <param name="antallFiler">AntallFiler</param>
+        /// <param name="basisregistrering">Basisregistrering</param>
+        /// <param name="xmlns">Xmlns</param>
```

Comment:
These comments aren't that descriptive. Do we have a better description of their content?